admin.php

```diff
@@ -7,13 +7,34 @@
  
 
 }
-
+if($_SESSION['isadmin']==0)
+{
+  header('location: user.php');
+  exit();
+}
 
 
 ?>
+ <?php
+$db = mysqli_connect('localhost', 'root', 'java@123', 'sanjeet');
+if ( isset($_GET['status'])){
+  if($_GET['status']==0){
+    $update ="UPDATE `admin_account` SET `Is_active`='1' WHERE id=". $_GET['edit_id'];
+    $result = mysqli_query($db, $update);
+    header('location: admin.php');
+
+  }
+  else{
+    $update ="UPDATE `admin_account` SET `Is_active`='0' WHERE id=". $_GET['edit_id'];
+    $result = mysqli_query($db, $update);
+    header('location: admin.php');
+  }
+}
+
+?> 
 
 <?php
-$db = mysqli_connect('localhost', 'phpmyadmin', 'java@123', 'admin');
+$db = mysqli_connect('localhost', 'root', 'java@123', 'sanjeet');
```

Comment:
same as above, don't create db connection again and again. The connection shoul be created only once.